spec/features/idv/flow_spec.rb

```diff
@@ -347,7 +347,9 @@
     end
 
     scenario 'pick USPS address verification' do
-      sign_in_and_2fa_user
+      user = sign_in_and_2fa_user
+      loa3_sp_session
```

Comment:
Oh. I didn't realize that. If it's exactly the same test, then yes, we only want the "live" one.